jobbergate-api/jobbergate_api/apps/job_scripts/routers.py

```diff
@@ -97,7 +99,9 @@ async def job_script_create_from_template(
     if len(missing_keys) > 0:
         raise HTTPException(
             status_code=status.HTTP_400_BAD_REQUEST,
-            detail=(f"The template files are missing required files: {missing_keys} for {id_or_identifier}"),
+            detail=("The following required files are missing for job template {}: {}").format(
```

Comment:
I'm wondering if the extra parens are necessary?

```suggestion
            detail="The following required files are missing for job template {}: {}".format(
```

Reply:
Good catch, solved in [c06c77a](https://github.com/omnivector-solutions/jobbergate/pull/450/commits/c06c77a365b850f51b4a65abe680dcfac9c2cce9)